7E3.json

```diff
@@ -0,0 +1,4473 @@
+{
+  "Version": 79,
```

Comment:
Metadata for observed data is not optimally set
![grafik](https://user-images.githubusercontent.com/25061876/215835001-ca14a76c-b4fd-4b1c-b3a7-7119aeb897ca.png)

I assume the molecule should be **7E3** ? Also Species (Mouse?) and Compartment (if applies) could be added.
This could be done simultaneously for multiselected datasets
![grafik](https://user-images.githubusercontent.com/25061876/215835816-bde9efcd-ec03-4c52-8aad-d8233abf71b9.png)

Also **Tissue** should be renamed to **Organ** (this **cannot** be done simultaneously)

Reply:
added the metadata now. The "7000" is a bug, see #2553 ("7000" is not in the .json file)